packages/react-components/tests/tree-button-group.test.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { TreeButtonGroup } from '../lib';
+import { render, fireEvent } from '@testing-library/react';
+
+test('It disabled buttons', () => {
+  const root = render(
+    <TreeButtonGroup disableClear={true} disableReset={true} disableRestore={true} />,
+  );
+  expect(
+    root.container.querySelector('button#clear-button')?.hasAttribute('disabled'),
+  ).toBeTruthy();
+  expect(
+    root.container.querySelector('button#reset-button')?.hasAttribute('disabled'),
+  ).toBeTruthy();
+  expect(
+    root.container.querySelector('button#restore-button')?.hasAttribute('disabled'),
+  ).toBeTruthy();
+});
+
+test('It executes callbacks correctly', () => {
+  const handleResetClick = jest.fn();
+  const handleClearClick = jest.fn();
+  const handleRestoreClick = jest.fn();
+  const root = render(
+    <TreeButtonGroup
+      handleResetClick={handleResetClick}
+      handleClearClick={handleClearClick}
+      handleRestoreClick={handleRestoreClick}
+    />,
+  );
+
+  const clearButton = root.container.querySelector('button#clear-button');
```

Comment:
RTL encourages use of the provided selectors when possible, in this case we could use `getByText`.

Reply:
Yeah, you're right! Thanks for the comment!